bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -4138,6 +4420,9 @@ ReplicaImp::ReplicaImp(const LoadedReplicaData &ld,
       }
 
       if (e.getForceCompleted()) seqNumInfo.forceComplete();
+      if (e.getIsExecuted()) ConcordAssert(e.getPrePrepareMsg()->seqNumber() > lastExecutedSeqNum);
+      vectorMapOfRecoveredRequests.push_back(std::make_pair<bool, Bitmap>(e.getIsExecuted(), e.getRequestsMap()));
+      if (e.getIsExecuted()) recoveringFromExecutionOfRequests = true;
```

Comment:
Shouldn't just be with the if 2 lines above?

Reply:
fixed